_docs/principles/accessibility.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Accessibility
+layout: docs
+category: Principles
+usage: >
+  Visual accessibility standards enable users with varied levels of color impairment to successfully navigate and use a Helix based UI.
+status: Stable
+resource: true
+last-modified: 2017-10-24
+---
+
+
+## Accessibility for visual design
+
+<div class="hxRow">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+Rackspace products must be made accessible for everyone – regardless of a person’s abilities. We can assume from the statistics that many of our users have some form of visual impairment. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that: 
+
+- 4.5% of the global population experience color blindness (1 in 12 men and 1 in 200 women)
+- 4% suffer from low vision (1 in 30 people)
+- 0.6% are blind (1 in 188 people)
+ 
+By using the following best practices, you can help ensure your design is accessible for the visually impaired.
+
+### Do not rely on color alone to convey meaning
+
+Color deficiency is a common issue faced in the design process. The most common form of color blindness is known as red and green color deficiency and most color deficient people suffer from either Protanopia (less sensitivity to red light) or Deuteranopia (less sensitivity to green light). Red and green color deficiency can mean that those affected have troubles distinguishing certain hues of red and green. For example, an individual with red or green color deficiency can confuse blue and purple as they may not be able to distinguish the red element of purple.
+
+
+{% endcolumn %}
+
+{% column right:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+{% figure [caption:"Helix colors as seen in normal vision _(left)_; Protanopia - reduced sensitivity to red _(center)_; Deuteranopia - reduced sensitivity to green _(right)_."] [class:"image bg-light border"] %}
```

Comment:
This is a nit pick, but I think this set of images could read much better split into three figures, with their respective captions under each one:

[Figure 1]
Caption: Helix colors as seen in normal vision

[Figure 2]
Caption: Protanopia - reduced sensitivity to red

[Figure 3]
Caption: Deuteranopia - reduced sensitivity to green

Reply:
Updated.